base/bucket.go

```diff
@@ -32,32 +32,68 @@ const (
 )
 
 const (
-	GoCouchbase CouchbaseDriver = iota
-	GoCB
-	GoCBGoCouchbaseHybrid // Defaults to GoCB, falls back to GoCouchbase for missing functionality
+	GoCouchbase            CouchbaseDriver = iota
+	GoCB                                   // Use GoCB driver with default Transcoder
+	GoCBCustomSGTranscoder                 // Use GoCB driver with a custom Transcoder
 )
 
 const (
 	DataBucket CouchbaseBucketType = iota
 	IndexBucket
 )
 
-var (
-	DefaultDriverForBucketType = map[CouchbaseBucketType]CouchbaseDriver{
-		DataBucket: GoCBGoCouchbaseHybrid,
-		// DataBucket:  GoCouchbase,
-		IndexBucket: GoCB,
-	}
+const (
+	ViewQueryParamStale         = "stale"
+	ViewQueryParamReduce        = "reduce"
+	ViewQueryParamStartKey      = "startkey"
+	ViewQueryParamEndKey        = "endkey"
+	ViewQueryParamInclusiveEnd  = "inclusive_end"
+	ViewQueryParamLimit         = "limit"
+	ViewQueryParamIncludeDocs   = "include_docs"
+	ViewQueryParamDescending    = "descending"
+	ViewQueryParamGroup         = "group"
+	ViewQueryParamSkip          = "skip"
+	ViewQueryParamGroupLevel    = "group_level"
+	ViewQueryParamStartKeyDocId = "startkey_docid"
+	ViewQueryParamEndKeyDocId   = "endkey_docid"
+	ViewQueryParamKey           = "key"
+	ViewQueryParamKeys          = "keys"
```

Comment:
I probably looked for a `base/views.go` and didn't see one.  I created one in 291c6297e6bbeba8976dbf7a985f111767be3177 and moved those constants over there.